js/main.js

```diff
@@ -122,6 +131,12 @@ function showEntriesList() {
   $createEntry.setAttribute('class', 'view hidden');
   data.view = 'entries';
   data.editing = null;
+  $imgEntry.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $formElement.reset();
+  $formTitle.textContent = 'New Entry';
+  $deleteAnchor.remove();
```

Comment:
Instead of deleting the element, I suggest you just hide and show it as necessary.